ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/base/v1/base/clientlibs/editor/js/editDialog.js

```diff
@@ -79,6 +81,19 @@
         }
     }
 
+    /**
+     * Prefills all the multifield values in enum Names and deletes the hidden input
+     * @param {HTMLElement} dialog The dialog on which the operation is to be performed.
+     */
+    function prefillEnumNames(dialog) {
+        var visibleEnumNames = dialog.find(BASE_ENUMNAMES_VISIBLE);
+        var hiddenEnumNames = dialog.find(BASE_ENUMNAMES_HIDDEN);
+        for (var i = 0; i < hiddenEnumNames.length; i++) {
```

Comment:
pls put null checks.

Reply:
it will never be null, it will have length 0.
<img width="904" alt="Screenshot 2022-09-16 at 1 13 41 PM" src="https://user-images.githubusercontent.com/47965724/190584571-58ee50eb-1247-44ef-b347-eb29a73c7619.png">

Reply:
ack. thanks.